### PR TITLE
feat: add block option drop down

### LINKED ISSUE #474: [FEATURE] add block option drop down
### Description

add this code
```javascript
<Dropdown.Item
    onClick={() => console.log('blocking')}>
      Block
</Dropdown.Item>
``` 

after the quote reply
https://github.com/Dun-sin/Whisper/blob/6fdc60f6eab5430b1d86df943ecad3626e1d94b2/client/src/components/Chat.jsx#L558-L586

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by Anushka-Bhowmick**
# Fixes #474 

**My PR closes #474**

# 👨‍💻 Changes proposed(added an onClick code snippet to enhance the code

# ✔️ Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots

### CODE REVIEW

**Comment by Dun-sin** [comment]
> # PR Compliance Checks
> 
> Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project. The results are listed in the following section.
> ## Issue Reference In order to be considered for merging, the pull request description must refer to a specific issue number. This is described in our [Contributing Guide](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md). This check is looking for a phrase similar to: "Fixes #XYZ" or "Resolves #XYZ" where XYZ is the issue number that this PR is meant to address.
> ## Conventional Commit PR Title
> 
> In order to be considered for merging, the pull request title must match the specification in [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). You can edit the title in order for this check to pass. Most often, our PR titles are something like one of these:
> 
>     * docs: correct typo in README
> 
>     * feat: implement dark mode"
> 
>     * fix: correct remove button behavior
> 
> 
> Linting Errors
> 
>     * Found type "null", must be one of "feat","fix","docs","style","refactor","perf","test","build","ci","chore","revert"
> 
>     * No subject found

@Anushka-Bhowmick 
Please fix this error, if you don't know how, try this:
- Read the error carefully or read the contributing file
- Checkout past merged Pull requests to see how they were done for it to be merged

> Note: you can edit your PR message, this PR won't be merged if the error isn't gone and you have a week max to fix it before the PR gets closed without being merged

**Comment by Dun-sin** [comment]
> # PR Compliance Checks
> 
> Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project. The results are listed in the following section.
> ## Issue Reference In order to be considered for merging, the pull request description must refer to a specific issue number. This is described in our [Contributing Guide](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md). This check is looking for a phrase similar to: "Fixes #XYZ" or "Resolves #XYZ" where XYZ is the issue number that this PR is meant to address.
> ## Conventional Commit PR Title
> 
> In order to be considered for merging, the pull request title must match the specification in [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). You can edit the title in order for this check to pass. Most often, our PR titles are something like one of these:
> 
>     * docs: correct typo in README
> 
>     * feat: implement dark mode"
> 
>     * fix: correct remove button behavior
> 
> 
> Linting Errors
> 
>     * Found type "enhancement", must be one of "feat","fix","docs","style","refactor","perf","test","build","ci","chore","revert"

@Anushka-Bhowmick still there, you know what to do already

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/components/Chat.jsx [hunk 1]
```diff
@@ -584,6 +584,15 @@ const Chat = () => {
...
                                                     </Dropdown>
+                                                    <Dropdown.Item
+                                                       onClick={() =>
+                                                          console.log(
+                                                            'blocking'
+                                                              )
+                                                            }
+                                                        >
+                                                            Block
+                                                    </Dropdown.Item>
```
[the remark this answers is not in the document]
Comment (Anushka-Bhowmick) [reply client/src/components/Chat.jsx]: updated it

**Review by Dun-sin — CHANGES_REQUESTED** [review]
wrong again, after quote reply means after the dropdown.item for quote reply, when you make a change like this, ask yourself, does this make sense? You know the answer is no

File: client/src/components/Chat.jsx [hunk 2]
```diff
@@ -582,16 +582,16 @@ const Chat = () => {
...
                                                         </Dropdown.Item>
+                                                        <Dropdown.Item
+                                                             onClick={() => 
+                                                                console.log(
+                                                                    'blocking'
+                                                                    )
+                                                                }
+                                                            >
+                                                        Block
+                                                    </Dropdown.Item>
```
Comment (Dun-sin) [inline client/src/components/Chat.jsx:594]: looks like you are doing this directly from GitHub because it's not properly formatted

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/components/Chat.jsx (hunk 2 above)
Comment (Anushka-Bhowmick) [reply client/src/components/Chat.jsx:594]: No, i'm using VS Code but ig prettier isn't working properly. I'm checking it.

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Comment by Anushka-Bhowmick** [comment]
Thank you! Learned quite some stuffs :)

**Pull request merged**